doc/COMPILING/COMPILING-VS-VCPKG.md

```diff
@@ -30,7 +30,7 @@ Steps from current guide were tested on Windows 10 (64 bit), Visual Studio 2019
 In a `cmd.exe` shell:
 ```cmd
 REM cd to the appropriate folder first
-git clone https://github.com/Microsoft/vcpkg.git
+git clone --depth 1 https://github.com/Microsoft/vcpkg.git
```

Comment:
This may potentially trip people up if they clone and the tip commit is broken and we ask them to reset back in history but they can't because they haven't fetched it. We'll just have to pay attention to future help requests.